app-backup/buttersink/buttersink-0.6.8.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="rsync-like utility for btrfs snapshots"
+HOMEPAGE="https://github.com/AmesCornish/buttersink"
+
+LICENSE="GPL-3"
+SLOT=0
+
+# tests require network access
+RESTRICT="test"
+
+if [[ "${PV}" == 9999 ]] ; then
```

Comment:
I'm not a machine, I don't always notice everything ;-P. Plus, I usually leave suggestion on tiny thingies only when it needs changes anyway.